tests/e2e/tests/critical-path/notifications/notification-center.e2e.ts

```diff
@@ -67,6 +75,10 @@ test('Verify that user can open notification center by clicking on icon and see
         await t.expect(notificationPage.notificationTitle.withExactText(jsonNotifications[i].title).exists).ok('Displayed title');
         await t.expect(notificationPage.notificationBody.withExactText(jsonNotifications[i].body).exists).ok('Displayed body');
         await t.expect(notificationPage.notificationDate.withExactText(await notificationPage.convertEpochDateToMessageDate(jsonNotifications[i])).exists).ok('Displayed date');
+        if (!jsonNotifications[i].category !== undefined) {
```

Comment:
can you please add comment what checked here? for readability

Reply:
Seems like no explanation needed here. We just have a conditions that we can work with category only in case if it exists in the notification. Don't you think so?